Tasks/VsTestV2/README.md

```diff
@@ -34,7 +34,7 @@ Use the following options to select tests and control how the tests are run
 
 - **Path to Custom Test Adapters:** Path to the testadapter for the framework in which the specified tests are written. Provided directory and all subdirectories are checked for testadapters. If there is a packages folder in the sources directory, it is automatically searched for testadapters. As a result, any testadapter downloaded as a Nuget package will be used without any input. For example, ‘$(Build.SourcesDirectory)\Fabrikam\packages’
 
-- **Other console options:** Other options that can be provided to vstest.console.exe. For example, if you are using VSIX extensions, you can provide “/UseVsixExtensions:true”
+- **Other console options:** Other options that can be provided to vstest.console.exe. For example, if you are using VSIX extensions, you can provide “/UseVsixExtensions:true”. This setting is supported only when the phase parallelism is not set to multi-agent, and the task is configured to select tests based on test assemblies and run with the default batching option.
 
```

Comment:
the sentence mixes positive and negative conditions and is a bit confusing to read.

Let's re-use what we have in the documentation: 
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/vstest?view=vsts#arguments

Reply:
Please don't forget to add that "In these cases, the options can be specified using a runsettings file instead."